lib/rollbar/delay/resque.rb

```diff
@@ -23,7 +23,12 @@ def self.perform(payload)
         end
 
         def perform(payload)
-          Rollbar.process_payload_safely(payload)
+          begin
+            Rollbar.process_payload_safely(payload)
+          rescue
+            # Raise the exception so Resque can track the errored job
```

Comment:
Is this begin-rescue necessary? Seems like the code would function the same if it was just:

```ruby
def perform(payload)
  Rollbar.process_payload_safely(payload)
end
```